package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-loop",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
does your app break if you upgrade to the latest v3 (3.1)? I think it should be backwards compatibile. 3.0 was breaking but 3.1 brought back compatiblity with v2 (will be removed in v4).  If it's not actually backwards compatible  then we can do v2 also. if it is though,, then we only need 3.1